src/index.html

```diff
@@ -1,13 +1,59 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__image image">
+        <img
+          src="./images/imac.jpeg"
+          alt="Mononblock APPLE A1419 iMac 27 Retina 5K"
+          class="image__product"
+        >
+      </div>
+
+      <div class="card__info">
+        <h2 class="card__info-name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__info-code">Product code: 195434</p>
+      </div>
+
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__index">Reviews: 5</p>
+      </div>
+
+      <div class="card__price price">
+        <p class="price__txt">Price:</p>
+        <p class="price__check">$2,199</p>
+      </div>
+
+      <button class="card__button button">
+        <a
```

Comment:
Consider using a button OR link element depending on the functionality.
There is no need to use both